executor_bot/src/quote.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  Connection,
+  PublicKey,
+  Keypair,
+  AccountMeta,
+  Cluster,
+} from "@solana/web3.js";
+import * as Poseidon from "@mithraic-labs/poseidon";
+import { Jupiter } from "@jup-ag/core";
+import {
+  getMint,
+  AccountLayout,
+  getOrCreateAssociatedTokenAccount,
+} from "@solana/spl-token2";
+import JSBI from "jsbi";
+import { JUPITER_EXCLUDED_AMMS, ONLY_DIRECT_ROUTE } from "./constants";
+import { openbookData, OPENBOOK_V3_PROGRAM_ID, raydiumTradeAccts } from "./dex";
+import { Market } from "@project-serum/serum";
+import config from "./config";
+import { wait } from "./utils";
+
+export const getQuote = async ({
+  boundedStrategy: {
+    collateralAccount,
+    collateralMint,
+    depositAddress,
+    boundedPriceNumerator,
+    boundedPriceDenominator,
+  },
+  connection,
+  payer,
+}: {
+  boundedStrategy: Poseidon.BoundedStrategyV2;
+  connection: Connection;
+  payer: Keypair;
+}) => {
+  let remainingAccounts = [] as AccountMeta[];
+  let additionalData = [];
+  const [collateralAccountBuff, depositAccountBuff] =
```

Comment:
nit: Could reduce RPC calls by loading the mint data in this `getMultipleAccountsInfo` and then deserializing